src/replication.c

```diff
@@ -2160,136 +2182,131 @@ void syncWithMaster(connection *conn) {
             strncmp(err,"-ERR operation not permitted",28) != 0)
         {
             serverLog(LL_WARNING,"Error reply to PING from master: '%s'",err);
-            sdsfree(err);
+            sdsfree(err), err = NULL;
```

Comment:
So is this an official style now? 🙄

Reply:
if we'll take a vote i'll say "yes". but if you think i'm wrong to start introducing these in random places in redis, i'm willing to cave.

the reason i did that in this case is since i consider these two a "single statement".
so although that's not needed can i can `;` instead of `,`, this lets me use a single line, which will also make sure people can't later relocate one of these without the other.